README.md

```diff
@@ -253,6 +253,29 @@ Promise.all([builder.trace('app/first.js'), builder.trace('app/second.js')])
 });
 ```
 
+### Using the compiler cache
```

Comment:
With new features like this I prefer not to document them in code over the readme, so that we aren't stuck with the API just yet.